CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+## v4.0.2
```

Comment:
Hi @Ikana! Basically in your terminal you can run the following steps:
```
git checkout master
git pull airbnb master
git checkout fix-get-client-bounding-rect
git rebase master
git push -f origin fix-get-client-bound-rect
```

If you don't have airbnb configured as a remote branch, you'll have to do:
```
git remote add airbnb git@github.com:airbnb/react-dates.git
```